vaadin-spring/src/main/java/com/vaadin/flow/spring/security/VaadinWebSecurityConfigurerAdapter.java

```diff
@@ -75,6 +75,8 @@ protected void configure(HttpSecurity http) throws Exception {
                 // Vaadin internal requests must always be allowed to allow public Flow pages
                 // and/or login page implemented using Flow.
         urlRegistry.requestMatchers(requestUtil::isFrameworkInternalRequest).permitAll();
+        // Public endpoints are OK to access
+        urlRegistry.requestMatchers(requestUtil::isAnonymousEndpoint).permitAll();
```

Comment:
Totally out of scope.. do you think it would be worth to try something like `VaadinFusionWebSecurityConfigurerAdapter extends VaadinWebSecurityConfigurerAdapter` - this could move the `EndpointUtil` to the fusion specific class as well and would reduce the complexity in `VaadinWebSecurityConfigurerAdapter`.

Reply:
I think we want to have one security config that works for Flow, Fusion and hybrid apps